roles/ldap_services/templates/checksum.sh.j2

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+slapcat -H ldaps:///{{ services_ldap.basedn }} -a '(!(entryDN:={{ services_ldap.basedn }}))' 2>/dev/null | sort | sha1sum | awk '{ print $1 }'
```

Comment:
Deze sort lijkt me toch wat hackish; het zal meestal wel goed gaan, maar er zijn wel realistische scenario's waar een change niet in de sorted output doorkomt (bv gebruiker die van groep verandert).  

Nu lijkt Google idd niks zinnigs kant-enklaars op te leveren, dus ik heb mezelf maar even uitgeleefd.  Dit hekseltje lijkt te doen wat we willen: https://gist.github.com/baszoetekouw/0436704c28066919e88a9359c8b8e884

Kun jij dat eens testen?